src/lib/onHidden.ts

```diff
@@ -34,4 +33,7 @@ export const onHidden = (cb: OnHiddenCallback, once?: boolean) => {
   // Some browsers have buggy implementations of visibilitychange,
   // so we use pagehide in addition, just to be safe.
   addEventListener('pagehide', onHiddenOrPageHide, true);
+
+  // if the document is already hidden
+  onHiddenOrPageHide({});
```

Comment:
Adding this will cause problem for other functions that use `onHidden`. We do not want to call the callback immediately—only when the page is actually hidden.

I see that your changes to the `visibilityWatcher.ts` file account for calling it right away, but other modules import this function as well, and it won't work in many of those cases.